### PR TITLE
trivial:typo fix and add coverage.txt to gitignore

### PR DESCRIPTION
**Pull request by yyforyongyu**
Typo found when working on other issues, also added the `coverage.txt` to `.gitignore`.

### CODE REVIEW

File: lntest/node.go [hunk 1]
```diff
@@ -123,7 +123,7 @@ type BackendConfig interface {
 	// ConnectMiner is called to establish a connection to the test miner.
 	ConnectMiner() error
 
-	// DisconnectMiner is called to bitconneeeect the miner.
```
Comment (cfromknecht) [inline lntest/node.go]: lmao 😂

File: brontide/noise.go [hunk 2]
```diff
@@ -744,7 +743,7 @@ func (b *Machine) WriteMessage(p []byte) error {
 // returned reflects the number of plaintext bytes in the payload, and does not
 // account for the overhead of the header or MACs.
 //
-// NOTE: It is safe to call this method again iff a timeout error is returned.
+// NOTE: It is safe to call this method again if a timeout error is returned.
```
Comment (cfromknecht) [inline brontide/noise.go]: in this case, `iff` is correct as it means if-and-only-if

File: .gitignore [hunk 3]
```diff
@@ -63,3 +63,6 @@ profile.tmp
 .DS_Store
 
 .vscode
+
+# Coverage test
+coverage.txt
```
Comment (cfromknecht) [inline .gitignore]: we can probably split this off into its own commit, e.g.
 * `gitignore: add coverage.txt`
 * ` multi: fix typos`

File: lntest/node.go (hunk 1 above)
Comment (yyforyongyu) [reply lntest/node.go]: hahaha

File: brontide/noise.go (hunk 2 above)
Comment (yyforyongyu) [reply brontide/noise.go]: gotya

File: .gitignore (hunk 3 above)
Comment (yyforyongyu) [reply .gitignore]: fixed

File: lntest/node.go (hunk 1 above)
Comment (halseth) [reply lntest/node.go]: Doesn't look like a typo to me 😜

Comment (Roasbeef) [reply lntest/node.go]: lololol
